math/wxMaxima/Portfile

```diff
@@ -1,112 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem          1.0
 
 PortGroup           active_variants 1.1
 PortGroup           wxWidgets 1.0
 PortGroup           github 1.0
 
+github.setup        wxMaxima-developers wxmaxima 20.04.0 Version-
 name                wxMaxima
-version             18.02.0
-maintainers         nomaintainer
+maintainers         @MSoegtropIMC
```

Comment:
Can `openmaintainer` be added here? This is preferred for non-Macports committers so that we can fix various issues more easily without waiting for approval.

Reply:
I thought about that but at least initially I would prefer to be asked before things are changed I am now responsible for. I am almost always responsive in a shorter timeframe than this PR took. In case there are issues, we can still change this. As you have seen I have some strong opinions about details which others might think are unimportant.

> Thank you for your patience!

I say thank you for your detailed reviews - I learned a lot!